src/dlr.cc

```diff
@@ -70,35 +66,41 @@ extern "C" int GetDLRInputSizeDim(DLRModelHandle* handle, int index,
   API_END();
 }
 
-extern "C" int GetDLRWeightName(DLRModelHandle* handle, int index,
-                                const char** weight_name) {
+extern "C" int GetDLRWeightName(DLRModelHandle* handle, int index, const char** weight_name) {
   API_BEGIN();
   DLRModel* model = static_cast<DLRModel*>(*handle);
   CHECK(model != nullptr) << "model is nullptr, create it first";
   *weight_name = model->GetWeightName(index);
   API_END();
 }
 
-extern "C" int SetDLRInput(DLRModelHandle* handle, const char* name,
-                           const int64_t* shape, const void* input, int dim) {
+extern "C" int SetDLRInput(DLRModelHandle* handle, const char* name, const int64_t* shape,
+                           const void* input, int dim) {
   API_BEGIN();
   DLRModel* model = static_cast<DLRModel*>(*handle);
   CHECK(model != nullptr) << "model is nullptr, create it first";
   model->SetInput(name, shape, input, dim);
   API_END();
 }
 
-extern "C" int GetDLRInput(DLRModelHandle* handle, const char* name,
-                           void* input) {
+extern "C" int SetTVMInputTensor(DLRModelHandle* handle, const char* name, void* dltensor) {
+  API_BEGIN();
```

Comment:
Can you add a check here and to other new TVM model specific functions to make sure DLRBackend == kTVM?

Reply:
done, used strcmp since GetBackend returns a const char*. Currently theres no API get a `DLRBackend` enum value. If we decide to change this I can update the check to use that